media/js/firefox/new/scene2.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// set up namespace
+if (typeof Mozilla == 'undefined') {
+    var Mozilla = {};
+}
+
+Mozilla.Firefox = window.Mozilla.Firefox || {};
+
+;(function($, dataLayer, MFNew) {
+    'use strict';
+
+    var $directDownloadLink = $('#direct-download-link');
+    var $stage = $('#stage');
+    var downloadStarted = Number(MFNew.params.get('dlstarted')) === 1;
```

Comment:
Curious any reason you favor `Number` here over `parseInt` (which i guess we use more commonly)?